docker-for-mac/apple-m1.md

```diff
@@ -12,13 +12,13 @@ Welcome to Docker Desktop for Apple Silicon.
 >
 > We encourage you to try the release candidate and report any issues in the [Docker Desktop for Mac GitHub](https://github.com/docker/for-mac) repository.
 
-## Docker Desktop RC 1
+## Docker Desktop RC 2
 
-2021-03-18
+2021-03-25
 
 Click on the following link to download the latest release candidate of Docker Desktop for Apple Silicon.
```

Comment:
Insert:

This build allows users to switch between qemu-based and hypervisor.framework-based virtual machines. The default is qemu, but it can be switched back to hypervisor.framework in the experimental tab of the Preferences.

This is still a tech preview, and we are not committing to include either of these specific back ends for the final release. Which we release eventually will depend on user feedback. We hope that the new qemu back end will avoid some of the known issues listed below, and we would welcome confirmation of this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome reports of problems with it, especially areas where it is inferior to the hypervisor.framework.